firebase: add page (#4420)
tldr-pages/tldr: pages/common/firebase.md

# firebase

> Firebase Command Line Interface (CLI) Tools.

> > Used to test, manage, and deploy your Firebase project from the command line. More information: <https://www.npmjs.com/package/firebase-tools>.

Reviewer: I think the offical repository is better because you can install this tool without npm.
```suggestion
> Used to test, manage, and deploy your Firebase project from the command line.
> More information: <https://github.com/firebase/firebase-tools>.
```

- Login:

`firebase login`

> - Help:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display the help:
```

`firebase help`

> - List your firebase projects:

Reviewer 2: Perhaps this will read better?
```suggestion
- List existing firebase projects:
```
We generally try to  avoid the 2nd person in tldr pages :-)

`firebase project:list`

- Setup a firebase in the current directory:

`firebase init {{options}} {{feature}}`

> - Deploy code and assets to your project:

Reviewer 2: Perhaps this will read better?
```suggestion
- Deploy code and assets to the project in the current directory:
```
